assets/docs/tulasiram58827/keras-ocr/dr/1.md

```diff
@@ -1,11 +1,10 @@
 # Lite tulasiram58827/keras-ocr/dr/1
-
 TF Lite quantized version of the Keras OCR Model for text recognition from images.
 
 <!-- parent-model: tulasiram58827/keras-ocr/1 -->
-<!-- asset-path: https://github.com/tulasiram58827/ocr_tflite/releases/download/v0.1/keras_ocr_dr.tar.xz -->
+<!-- asset-path: https://github.com/tulasiram58827/ocr_tflite/releases/download/v0.2/keras_ocr_dr_ctc.tar.xz -->
```

Comment:
Models on tfhub.dev are immutable, so the correct way would be to create a version 2 of the model. 

To do so you need to create a 2.md with the current changes. 

Also, for better visual organization on tfhub.dev, I suggest to also add a second version (2.md) to the placeholder model (https://raw.githubusercontent.com/tensorflow/tfhub.dev/master/assets/docs/tulasiram58827/keras-ocr/1.md) and then use `<!-- parent-model: tulasiram58827/keras-ocr/2 -->` for the new Lite version.